security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+[id="cert-manager-operator-release-notes"]
+= {cert-manager-operator} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-release-notes
+
+toc::[]
+
+TODO: Quick overview
+
+:FeatureName: The {cert-manager-operator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+For more information about the {cert-manager-operator}, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
+
+// TODO: Link to upstream release notes too
+
+// TODO: Confirm 1.7.1 version
```

Comment:
@slaskawi Will 1.7.1 be the first release of the cert-manager operator?

Reply:
Yes!